tests/Writer/CSV/WriterTest.php

```diff
@@ -106,6 +108,32 @@ public function testWriteShouldSupportBooleanValues(): void
         self::assertSame('1,0', $writtenContent);
     }
 
+    public function testWriteShouldSupportDateTimeValues(): void
+    {
+        $now = new DateTime();
+        $options = new Options();
+        $allRows = $this->createRowsFromValues([
+            [$now],
+        ]);
+        $writtenContent = $this->writeToCsvFileAndReturnWrittenContent($allRows, 'csv_with_datetime_values.csv');
+        $writtenContent = $this->trimWrittenContent($writtenContent);
+
+        self::assertSame($now->format(DATE_ATOM), $writtenContent);
```

Comment:
In order for tests to be reliable, assertions should be done on hardcoded strings, so eventual bugs don't propagate.
Fix the date and time of the `DateTime` object and assert the output accordingly, doing something like:

```php
$now = new DateTime('2023-11-20 10:34:21');
// [...]
self::assertSame('2023-11-20T10:34:21+00:00', $writtenContent);
```

Reply:
Fixed